src/features/defi/providers/fox-farming/components/FoxFarmingManager/Overview/FoxFarmingOverview.tsx

```diff
@@ -135,7 +118,9 @@ export const FoxFarmingOverview: React.FC<FoxFarmingOverviewProps> = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [highestBalanceAccountId])
 
-  const stakingAsset = useAppSelector(state => selectAssetById(state, foxEthLpAssetId))
+  const stakingAsset = useAppSelector(state =>
+    selectAssetById(state, opportunityData?.underlyingAssetId ?? ''),
```

Comment:
```suggestion
    selectAssetById(state, opportunityData?.underlyingAssetId),
```

Reply:
Impossible with `selectAssetById` selector for now